src/test/java/com/auth0/net/BaseRequestTest.java

```diff
@@ -0,0 +1,122 @@
+package com.auth0.net;
+
+import com.auth0.exception.APIException;
+import com.auth0.exception.Auth0Exception;
+import com.auth0.exception.RateLimitException;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class BaseRequestTest {
```

Comment:
Please add a `FIXME` comment at the class level pointing to the link you shared in the issue. My idea here is that in the future when they let users use this feature programmatically, we can come back to this code and fix it. Because today it seems enabling this extension is affecting the whole tests folder and not just a single class.  https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods

Reply:
Good idea, will do.